sdk/python/feast/cli.py

```diff
@@ -684,6 +684,13 @@ def init_command(project_directory, minimal: bool, template: str):
     default=5,
     show_default=True,
 )
+@click.option(
+    "--metrics",
+    "-m",
+    is_flag=True,
+    show_default=True,
+    help="Enable the Metrics Server",
```

Comment:
what is the purpose of    ` is_flag=True,
    show_default=True`, sorry if Missed any conversation related to that